src/components/dialog-full-screen/dialog-full-screen.spec.js

```diff
@@ -44,6 +43,37 @@ describe('DialogFullScreen', () => {
     });
   });
 
+  describe('modalHTML', () => {
+    beforeEach(() => {
+      wrapper = mount(
+        <DialogFullScreen
+          open
+          className='foo'
+          title='my title'
+          onCancel={ onCancel }
+        >
+          <Button>Button</Button>
+          <Button>Button</Button>
+        </DialogFullScreen>
+      );
+      instance = wrapper.instance();
+    });
+
+    it('renders the dialog', () => {
+      expect(instance._dialog).toBeTruthy();
+    });
+
+    it('closes when the exit icon is click', () => {
```

Comment:
We should add the same tests that we added to `Dialog` (ESC, Enter, a)

Reply:
done.